packages/repository/src/__tests__/unit/model/model.unit.ts

```diff
@@ -313,11 +307,7 @@ describe('model', () => {
     Category.definition = new ModelDefinition('Category')
       .addSetting('strict', false)
       .addProperty('id', {type: 'number', id: true, required: true})
-      .addRelation(<HasManyDefinition>{
```

Comment:
Let's revert this part and the test at line 479, or there is no tests cover `addRelation`.